docs/management_api.md

```diff
@@ -285,6 +285,100 @@ curl http://localhost:8081/models/noop/all
 ]
 ```
 
+`GET /models/{model_name}/{model_version}?customized=true`
+or 
+`GET /models/{model_name}?customized=true`
+
+Use the Describe Model API to get detail runtime status and customized metadata of a version of a model:
```

Comment:
@lxning lets clarify here if custom handler extend the BaseHandler then there is no need to override the "_is_describe".